CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 23.12+ (???)
 ------------------------------------------------------------------------
+- Feature: [#1238]: Mountain tool is now a toggle, making it possible to specify the mountain table size.
+- Feature: [#2228]: Max terraform tool sizes have been increased from 10x10 to 64x64.
+- Change: [#2228]: The landscape 'paint mode' button now uses a different paintbrush image.
```

Comment:
Might be a bit much to include in the changelog. Thoughts?

Reply:
I certainly don't mind padding out the changelog, especially if they are separate changes like you have here.